src/include/elist.h

```diff
@@ -32,20 +34,28 @@ template<typename T>
 class elist {
 public:
   struct item {
+  private:
+    elist *_list = NULL;
```

Comment:
this increases the size of each element by 8 bytes. not sure it would be a pure win.

Reply:
Yeah. Currently only several places will use this and with an alternative we can iterate list and count it in a loop.

Reply:
my concern is the scale of the list. what is the size of the linked list in general?

Reply:
In mds for some use cases, like the CDir::dirty_dentries, the list size maybe several handreds in overload case, but the list will be trimed every tick.